config.py

```diff
@@ -1,20 +1,36 @@
+
 DEBUG = False
 
+
 NWIS_SITE_SERVICE_ENDPOINT = 'https://waterservices.usgs.gov/nwis/'
+NWIS_WATERPEAK_SITE_SERVICE_ENDPOINT = 'https://nwis.waterdata.usgs.gov/nwis/peak'
+
+
+### Hydrograph Event Config Vars ###
 
 # Site IDs available for charting. Expects a list of valid site IDs.
 SITE_IDS = []
 
 # Start and end of time series data #
```

Comment:
Maybe describe the variables as the start and end date of the flood event. They will be used for more than the hydrograph.